### DISCUSSION TITLE
Where can I find my deviceShare

### DISCUSSION THREAD

**Discussion by wekongsi** [Web3Auth/Web3Auth#1118] [Help]
Hi, I am using web3auth Plug and Play sdk and was trying to look for the device share but I couldn't find anything in the localStorage, it seems to me the localStorage tkey is an empty string but I can see there's a full private key in localStorage, could you point me to where the tkey share is being stored?

**Comment by shahbaz17** [comment]
Hey @wekongsi 

Please take a look at https://web3auth.io/docs/sdk/self-host
